gebru/roster.json

```diff
@@ -127,6 +127,10 @@
       "materials": [
         {
           "type": "cv"
+        },
+        {
+          "type": "video",
+          "url": "https://youtu.be/RBXXco5ldZs"
```

Comment:
@Dnayir 
I know the name of the key is misleading, but can you please replace the full url with the video_id
i.e, "url": "RBXXco5ldZs"

Reply:
all done, apologies for the delay